contracts/finance/andromeda-validator-staking/src/contract.rs

```diff
@@ -192,11 +260,12 @@ fn execute_unstake(
         amount: fund.clone(),
     });
 
-    let mut unstaking_queue = UNSTAKING_QUEUE.load(deps.storage).unwrap_or_default();
-    unstaking_queue.push(UnstakingTokens {
-        fund,
-        payout_at: Timestamp::default(),
-    });
+    let unstaking_queue = UNSTAKING_QUEUE
```

Comment:
What was the adjustment here for?

Reply:
Just defaulting to a vector with the desired info instead of an empty vector and then pushing into it the desired info.